content/en/docs/installation/upgrading/upgrading-0.13-0.14.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "Upgrading from v0.13 to v0.14"
+linkTitle: "v0.13 to v0.14"
+weight: 30
+type: "docs"
+---
+
+Due to changes in the Deployment selector you will need to remove the deployments first before being able to upgrade.
+
+If you are using a deployment tool that automatically handles this (i.e. Helm),
+there should be no additional action to take.
```

Comment:
Is this true?

Reply:
(that Helm handles it automatically)

Reply:
Gonna test this in alpha 1 now!

Reply:
Okay... I wasn't able to fully test it with alpha 0, alpha 1 now clearly fails... we have to add instructions for that

Reply:
Updated it, tested it on helm 2 and 3 and they seem fine with deleting the deployments